assignment/index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html lang="en-US">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
```

Comment:
Though the browser lets you get away with it, this is actually a React-only thing and not correct HTML.